JS_2_names.js

```diff
@@ -0,0 +1,21 @@
+var arrUserNames = [];
```

Comment:
Try to use Hungarian notation to explain expected type of the variable in JS:
`aUserNames` - array; `sMyName` - string, `iPower` - integer number; `fnOnLoadingSuccess` - function; `oPerson` - object

Reply:
done